.github/workflows/nightly-nodejs-validation.yml

```diff
@@ -25,16 +25,16 @@ jobs:
           'lts/*',   # latest lts
           'current'  # newest
         ]
-        include:
-          - runs-on: ubuntu-latest
-            node-version: '18.0'
-            # @typescript-eslint/eslint-plugin requires node ^18.18
-            # but we allow yarn install to run regardless to test the package
-            ignore-engines: 'true'
+        use-yarn-lock: ['true', 'false']
+    name: "Tests [Node.js ${{ matrix.node-version }}, ${{ matrix.runs-on }}, ${{ matrix.use-yarn-lock == 'true' && 'Using yarn.lock' || 'Re-creating yarn.lock'}}]"
     uses: ./.github/workflows/test.yml
     with:
       node-version: ${{ matrix.node-version }}
       runs-on: ${{ matrix.runs-on }}
+      use-yarn-lock: ${{ matrix.use-yarn-lock }}
+      # @typescript-eslint/eslint-plugin requires node ^18.18
+      # but we allow yarn install to run regardless to test the package
+      ignore-engines: ${{ matrix.node-version  == '18.0' && 'true' || 'false'}}
```

Comment:
Can we instead raise to `18.18` as our minimum supported `node-version` within the job matrix?